### PR TITLE
DSDEGP-1586 DSDEGP-1589 Bump caseapp version to pull in fixes.

### PR DESCRIPTION
**Pull request by danxmoran**
### Description

Fixes:
1. Missing options being reported as --camelCase instead of --kebab-case
2. Only one missing option being reported at a time

image

I didn't add tests because the functionality is coming from upstream (and it's tested there), but I can throw some in for good measure if people want.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: Ensure that you have added or updated tests
- [x] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala [hunk 1]
```diff
@@ -124,7 +125,7 @@ class ClioClient(webClient: ClioWebClient,
     * caseapp supports setting these through annotations, but
     * only if the values are constant strings.
     */
-  private val beforeCommandMessages: Messages[CommonOptions] =
+  private val beforeCommandMessages: Help[CommonOptions] =
```
Comment (pshapiro4broad) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala]: Should the name change too, e.g. `beforeCommandHelp`?

File: clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala [hunk 2]
```diff
@@ -130,6 +131,6 @@ object ClioCommand extends ClioParsers {
     * Sub-command-related messages to show when a users asks
     * for help or usage.
     */
-  val messages: CommandsMessages[ClioCommand] =
-    CommandsMessages[ClioCommand]
+  val messages: CommandsHelp[ClioCommand] =
```
Comment (pshapiro4broad) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala]: same comment here about `val` name

**Review by pshapiro4broad — APPROVED** [review]
Nice!

File: clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala [hunk 3]
```diff
@@ -103,25 +98,6 @@ object ClioCommand extends ClioParsers {
...
-   * run "show implicit parameters" on `CommandParser[ClioCommand]` below in
-   * IntelliJ without this import it displays a "no implicit found" error message).
-   *
-   * By importing the implicits from the companion object directly, we get
-   * induction to actually work. The `implicitly` call both demonstrates that
-   * the base-case implicit is properly in-scope and convinces IntelliJ that
-   * the import is actually used for something.
-   */
-  import caseapp.CommandParser._
-  implicitly[CommandParser[CNil]]
```
Comment (danxmoran) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala:123]: clean-compile from a cold SBT is working for me without this now :tada:

Somebody else might want to check that it works for them before we delete it.

**Review by tbl3rd — APPROVED** [review]

**Pull request merged**